_posts/en/newsletters/2022-04-06-newsletter.md

```diff
@@ -0,0 +1,238 @@
+  options can be used specify additional recipients, specify a subset of
+  the wallet's UTXO pool to be used as inputs, or to maximize the
+  recipient amount by skipping dust rather than leaving no UTXOs behind.
+  `sendall` therefore offers a convenient alternative to achieve some
+  applications of the `fSubtractFeeAmount` argument to other `send*`
+  RPCs, but `fSubtractFeeAmount` is still best option for paying a
+  recipient who is liable for
+  transaction fees.
+
+- [Bitcoin Core #23536][] sets validation rules to always enforce [taproot][topic taproot]
+  whenever segwit is enforced, including validation of historical blocks
+(except for block 692261 which included a transaction spending a v1 witness
+output before taproot activated). This change has also been done for P2SH and
+segwit soft forks (see [Newsletter #60][news60 buried]); it simplifies testing and code review, mitigates some
+risk of potential bugs in deployment code, and offers belt-and-suspenders
+protection for extreme scenarios in which a node downloads an alternative
+most-work block chain where taproot has not activated.
+
+- [Bitcoin Core #24555][] adds documentation for running Bitcoin Core over the
+  CJDNS [network][topic anonymity networks] (see [Newsletter #175][news175 cjdns]).
+
+- [C-Lightning #5013][] adds the ability to administer a node using gRPC
+  with mTLS authentication.
+
+- [C-Lightning #5121][] updates the `invoice` RPC with a new `deschash`
+  parameter that will include the hash of arbitrary data in the
+  [BOLT11][] invoice rather than a short description string.  This is
+  supported by schemes such as [LNURL][] that can send large
+  descriptions (including data like images) through a separate
+  communication channel without BOLT11's constraints.
+
+- [Eclair #2196][] adds a `channelbalances` API that lists the balance
+  of all the node's channels, including channels that have been
+  disabled.
+
+- [LND #6263][] adds support for [taproot][topic taproot] keypath spends
+  to LND's wallet, which may now be tested using the default
+  [signet][topic signet].
+
+- [Libsecp256k1 #1089][] renames the `secp256k1_schnorrsig_sign()`
+  function to `..._sign32()`, making it clearer that this function signs
+  32-byte messages (e.g. a SHA256 hash digest).  This is in contrast
+  to the `secp256k1_schnorrsig_sign_custom()` function which signs an
+  arbitrary-length message (see [Newsletter #157][news157 schnorrsig]
+  for additional discussion).
+
+- [Rust Bitcoin #909][] adds support for creating optimal taproot
+  scriptpath trees using [huffman coding][] by passing in the
+  probability that a particular script path will be used.
+
+- [LDK #1388][] adds default support for creating ECDSA signatures that
+  are smaller than average by finding a value that can be encoded more
+  compactly, a process previously implemented in the wallets for Bitcoin
+  Core and C-Lightning (see [Newsletters #8][news8 lowr] and
```

Comment:
I'm going to continue calling it C-Lightning at least until they update their README.md or start poking me (the same rule we used for updating Rust-Lightning to LDK).  My reasoning is that (1) they should figure out their own internal consistency before we're bothered to make a change and (2) the way I saw the name change advertised, it looked like a Blockstream marketing push rather than something coming from the **open source** C-Lightning developer community---I haven't heard of any dissent about the name change, but I don't want to assume it's a done deal until there's clear evidence it has community support (like a PR to their repo to update their docs with ACKs from non-Blockstream contributors).

Reply:
I chatted with Christian Decker about it yesterday, and it sounded to me like it was a long time coming, their codebase now has also Python and Rust parts via the plugins and they wanted to rebrand for a while. I agree that they should figure out their own internal consistency first, though.